### ISSUE TITLE
CNA with chromosomes/chromosome arms

### ISSUE THREAD

**Issue by sofiedemeyer** [cbg-ethz/COMPASS#11]
I am testing this software, and it is really good. Thanks!
I am wondering whether it would be possible to call the CNAs based on the full chromosomes or the chromosome arms? Amplicons or genes are sometimes to specific..

**Comment by e-sollier** [comment]
Thanks for the nice feedback!
COMPASS itself is agnostic to what kind of regions are used for CNA inference, so it's definitely possible to consider whole chromosome or chromosome arms as regions for CNA inference. I updated the preprocess_loom.py script to include the "chromosome" and "chromosome_arm" options for the --region argument. For chromosome_arm, you also need to give the position of the centromeres wth the --centromere argument, and I added two files centromeres_GRCh37.txt and centromeres_GRCh38.txt in the preprocessing directory.

Using chromosome (arms) for CNA inference might be more robust, but keep in mind that many CNAs in cancer do not affect whole chromosome (arms), so this might give weird results if only some of the genes located in a chromosome (arm) are affected by a CNA.
